### ISSUE TITLE
Native Block Editor integration

### ISSUE THREAD

**Issue by fabiankaegy** [johnbillion/query-monitor#720]
Currently, query monitor only lives within the regular wp admin bar, which is the correct place for it. However, with Gutenberg taking over more and more of the UI the admin bar is not always reachable anymore. In the post editor you can only reach it when disabling the "Fullscreen mode". In the site editor however there is no option to even do this. So it is currently impossible to invoke query monitor when working in the site editor. 

It would be really neat to use the slot fill system of the block editor to also render a Gutenberg native UI for the query monitor menu.

**Comment by johnbillion** [comment]
Yeah agreed, but this is low on my list of priorities. #483 needs to come first anyway.

**Comment by lmisch** [comment]
+1 for this. I wish I was able to debug own gutenberg blocks with query monitor. :)
